LICENSE.md

```diff
@@ -1,6 +1,7 @@
 The MIT License (MIT)
 
 Copyright (c) 2016 Roel Nieskens, https://pixelambacht.nl
+Copyright (c) 2018 Google LLC
```

Comment:
Why is Google getting the copyright here? :-)

Reply:
Thanks for the review. When contributing to open source projects, we're asked to add Google as an author.